playground/mocks/data/idp/idx/authenticator-verification-okta-verify-signed-nonce-loopback-with-enhanced-polling.json

```diff
@@ -0,0 +1,240 @@
+{
+  "stateHandle": "02im-3M2f6UXHgNfS7Ns7C85EKHzGaKw0u1CC4p9_r",
+  "version": "1.0.0",
+  "expiresAt": "2020-01-13T21:14:37.000Z",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "rel": [
+          "create-form"
+        ],
+        "relatesTo": [
+          "$.currentAuthenticator"
+        ],
+        "name": "challenge-poll",
+        "href": "http://localhost:3000/idp/idx/authenticators/poll",
+        "method": "POST",
+        "accepts": "application/vnd.okta.v1+json",
+        "refresh": 4000,
```

Comment:
One more comment from office hours review, would it be possible to decrease this refresh time to allow us to lower the wait time [here](https://github.com/okta/okta-signin-widget/pull/3285/files#diff-698534168a87ff567c25b350509b7261a3660769a3c5fd36b10adc3873731f39R73)?